### ISSUE TITLE
Compile error under linux and windows. Commit 30d4889

### ISSUE THREAD

**Issue by n31r0** [godotengine/godot#2826]
under Linux:
```
/usr/bin/ld: cannot find -liconv
```

under Windows:
```
 LINK : fatal error LNK1181: cannot open input file 'iconv.windows.tools.lib'
```

**Comment by reduz** [comment]
uh what is adding iconv?

**Comment by akien-mga** [comment]
That's a regression of 30d4889 which fixed the linking of iconv on OSX, needed to build the new pe_bliss dependency.

**Comment by akien-mga** [comment]
Fixed by b7a1599.
